frontend/src/component/common/Sticky/StickyProvider.tsx

```diff
@@ -0,0 +1,133 @@
+import { useState, useCallback, ReactNode, RefObject, useEffect } from 'react';
+import { StickyContext } from './StickyContext';
+
+interface IStickyProviderProps {
+    children: ReactNode;
+}
+
+export const StickyProvider = ({ children }: IStickyProviderProps) => {
+    const [stickyItems, setStickyItems] = useState<RefObject<HTMLDivElement>[]>(
+        [],
+    );
+    const [resizeListeners, setResizeListeners] = useState(
```

Comment:
Do we need this state? Can we directly observe and unobserve the elements?

Reply:
Same as https://github.com/Unleash/unleash/pull/5088#discussion_r1365363311